mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -171,20 +164,34 @@ table.mtp-table--review {
 .mtp-check-cell__actions {
   display: flex;
   align-items: center;
+
+  .button, .button-secondary {
+    margin-bottom: 1rem;
+
+    @include media(mobile) {
+      width: auto;
+    }
+  }
 }
+
 .mtp-check-cell__list-status {
   margin-left: 1rem;
 }
 
-.mtp-check-cell__list-status--my-list {
+.mtp-check-cell__list-status--my-list, .mtp-check-cell__list-status--text {
   display: inline-block;
-  margin-bottom: 1.6rem;
+  margin-bottom: 0.6rem;
 }
 
 .mtp-check-cell__list-status--my-list::before {
-	content: "✓";
-	font-size: 1.5em;
-	color: #00703c;
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  margin-top: -5px;
+  vertical-align: middle;
+  content: '';
+  background: file-url('added-to-my-list.png') no-repeat;
```

Comment:
As mentioned on the SVG, this feels too specific, would it be easier to reuse and find it we just call it `tick.svg` or `icon-tick.svg`?

Reply:
fair point. it's a custom colour and size compared to the original checkmark graphic i found. still, might as well generalise the name!